X584.bpr

```diff
@@ -83,14 +83,14 @@ CodePage=1251
 
 [Version Info Keys]
 CompanyName=
-FileDescription=
-FileVersion=1.0.0.0
-InternalName=
-LegalCopyright=
+FileDescription=Симулятор процессора К584ВМ1
```

Comment:
I believe it's better to use English here. What about "A simulator of K584VM1 processor"?